src/main/kotlin/no/nav/syfo/application/SenOppfolgingService.kt

```diff
@@ -41,13 +41,23 @@ class SenOppfolgingService(
     }
 
     fun publishUnpublishedKandidatStatus(): List<Result<SenOppfolgingKandidat>> {
```

Comment:
Kandidat-status-cronjobben publiserer nå både upubliserte kandidat og upubliserte vurderinger for kandidater.